<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="leftLayout">
      <item>
       <widget class="QLineEdit" name="expr_input" />
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="2">
         <widget class="QPushButton" name="button_6">
          <property name="text">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="button_7">
          <property name="text">
           <string>7</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="button_8">
          <property name="text">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="button_4">
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="button_0">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="button_9">
          <property name="text">
           <string>9</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="button_dot">
          <property name="text">
           <string>.</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="button_add">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="button_5">
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="button_1">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="button_multiply">
          <property name="text">
           <string>*</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="button_equals">
          <property name="text">
           <string>=</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="button_divide">
          <property name="text">
           <string>/</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="button_3">
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="button_2">
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="button_subtract">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>

   <item><layout class="QVBoxLayout" name="rightLayout"><item><widget class="QPushButton" name="buttonPlot"><property name="text"><string>&#1043;&#1088;&#1072;&#1092;&#1080;&#1082;</string></property></widget></item><item><widget class="QLineEdit" name="lineEdit_xMin"><property name="placeholderText"><string>xMin</string></property></widget></item><item><widget class="QLineEdit" name="lineEdit_xMax"><property name="placeholderText"><string>xMax</string></property></widget></item><item><widget class="QLineEdit" name="lineEdit_yMin"><property name="placeholderText"><string>yMin</string></property></widget></item><item><widget class="QLineEdit" name="lineEdit_yMax"><property name="placeholderText"><string>yMax</string></property></widget></item><item><widget class="QCustomPlot" name="plotWidget" /></item></layout></item></layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
 </widget>
 <resources />
 <connections />
</ui>
